actions/installer/README.md

```diff
@@ -0,0 +1,37 @@
+# slsa-verifier setup GitHub Action
+
+This action installs the SLSA verifier and adds it to your PATH.
+
+For more information about `slsa-verifier`, refer to [its documentation](https://github.com/slsa-framework/slsa-verifier#verification-of-provenance).
+
+For more information about SLSA in general, see [https://slsa.dev](https://slsa.dev).
+
+## Usage
+
+To install the latest version, use:
+
+```yaml
+uses: slsa-framework/slsa-verifier-installer
+```
+
+To install a specific version, use:
+```yaml
+uses: slsa-framework/slsa-verifier-installer
+  with:
+    verifier-version: v1.3.0
+```
+
+NB: Pinning this action to a relase tag or commit SHA will pin you to the latest version of `slsa-verifier` at the time of release.
```

Comment:
```suggestion
NB: Pinning this action to a release tag or commit SHA will pin you to the latest version of `slsa-verifier` at the time of release.
```

BTW: what does NB mean? consider expanding it if it's an acronym.

Reply:
It's Latin, so expanding the acronym would be kind of weird (https://en.wikipedia.org/wiki/Nota_bene). I've just replaced it with "Note: ..."